Dev suhail

NamrataSweya opened the pull request.

Ahmed-Suhail committed:
suhail - test commit

NamrataSweya committed:
Namrata-YieldApi

add post, get, put api

NamrataSweya committed:
Namrata_yieldApi

Add model yiledconfig

NamrataSweya committed:
Namrata_change scriptfile

add sample file

NamrataSweya committed:
Namrata_projectcleanup

remove authenticate value and data

NamrataSweya committed:
Namrata_project cleanup

remove valid value

NamrataSweya committed:
Delete .env.local

NamrataSweya committed:
namrata_projectclean

clean

NamrataSweya committed:
Namrata_userregistration

add new api registerwithorganization ,for new requaiment

NamrataSweya committed:
Pushed in branch developer add all changes

Add All changes usernew register, yieldconfig, aclmodule,aclmodule permission,permission gaurd

NamrataSweya committed:
Namrata_change invitation process

update invitaion and add new api for changeupdate password

NamrataSweya committed:
Testapi_permission_namrata

add invitation user,aclmodule,useraclpermission

NamrataSweya committed:
namrata_inviteAclupdatestatus

update working in inviteruseraccepte invitation permission status update

NamrataSweya committed:
api and ui change register

Ahmed-Suhail committed:
suhail - updated

Ahmed-Suhail committed:
Merge branch 'dev_suhail' into developer_nm

Ahmed-Suhail committed:
Merge pull request #1 from d-rec/developer_nm

yield and acls from namrata merged into ssuhails branch

NamrataSweya committed:
update migration query

NamrataSweya committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

Ahmed-Suhail committed:
Merge pull request #2 from d-rec/developer_nm

updated migration for user_role merging  commit

NamrataSweya committed:
update new migration

Ahmed-Suhail committed:
Merge pull request #3 from d-rec/developer_nm

update new migration

NamrataSweya committed:
newschemaupdate

Ahmed-Suhail committed:
Merge pull request #4 from d-rec/developer_nm

newschemaupdate

NamrataSweya committed:
new update

Ahmed-Suhail committed:
Merge pull request #5 from d-rec/developer_nm

new update

Ahmed-Suhail committed:
suhail - comented typescript errors

Ahmed-Suhail committed:
old commits from namrata

Ahmed-Suhail committed:
suhail - resolved conflicts

NamrataSweya committed:
orgzanization_table update

NamrataSweya committed:
new migration

NamrataSweya committed:
newdrop update

Ahmed-Suhail committed:
Merge pull request #6 from d-rec/developer_nm

Developer nm

NamrataSweya committed:
all_table_add_onefile

NamrataSweya committed:
new_update

NamrataSweya committed:
updateuserdto

NamrataSweya committed:
updat_ui_drec

Ahmed-Suhail committed:
suhail - updated errors log and reverted schema definitions

Ahmed-Suhail committed:
suhail - merged namrata branch

Ahmed-Suhail committed:
suhail - commented seed users and organizations and devices adding

Commit:
added build related files

Ahmed-Suhail committed:
suhail - fixed import

Ahmed-Suhail committed:
suhail - fixed import

Commit:
suhail - updated lock file

Commit:
suhail - updated lock file

Ahmed-Suhail committed:
suhail - fixed import issues

Ahmed-Suhail committed:
Merge branch 'dev_suhail' of https://github.com/d-rec/drec-origin into dev_suhail

Ahmed-Suhail committed:
suhail -namrata sent updated migration

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/energywebfoundation/drec-origin?pullRequest=94) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/energywebfoundation/drec-origin?pullRequest=94) before we can accept your contribution.<br/>**0** out of **3** committers have signed the CLA.<br/><br/>:x: Ahmed-Suhail<br/>:x: NamrataSweya<br/>:x: Ubuntu<br/><hr/>**Ubuntu** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/energywebfoundation/drec-origin?pullRequest=94) it.</sub>

NamrataSweya committed:
new schema update

NamrataSweya committed:
add user_role insert

Ahmed-Suhail committed:
suhail - circluar depnedancy issues resolve and few import fixes

Ahmed-Suhail committed:
Merge branch 'dev_suhail' of https://github.com/d-rec/drec-origin into dev_suhail

Ahmed-Suhail committed:
suhail - commented seed devices, org and users

Ahmed-Suhail committed:
suhail - added log for jwt expiration

Ahmed-Suhail committed:
suhail - added payload log

Ahmed-Suhail committed:
suhail - added tsignore and removed fields

Ahmed-Suhail committed:
suhail - added expiration time manually will remove

Ahmed-Suhail committed:
suhail - updated imports, updated csv processing methodology as previous was not reading the second file sent,

Ahmed-Suhail committed:
suhail - updated imports, updated csv processing methodology as previous was not reading the second file sent,

Ahmed-Suhail committed:
suhail - added dropzone for csv file and added tables and data fetching for jobs added

Ahmed-Suhail committed:
suhail - updated regex and route name

Ahmed-Suhail committed:
suhail - returns all orgnaization id jobs rather than only completed

Ahmed-Suhail committed:
suhail - added log

Ahmed-Suhail committed:
suhail - updated query wrong field

Ahmed-Suhail committed:
suhail - added consoles

NamrataSweya committed:
meterread and user registration

add new mreter read process(delta,Aggregate) and change orgadmin register status

Ahmed-Suhail committed:
suhail - updated permission acl for device

Ahmed-Suhail committed:
suhail - added UI workarounds for certificate geneartion

NamrataSweya committed:
meterread,reset Password,device list filter for buyer reservation

change the in meterread  api, devce filter api,user invite reset  password

Ahmed-Suhail committed:
suhail - added lock file

Ahmed-Suhail committed:
suhail - resolved conflicts

Ahmed-Suhail committed:
Merge pull request #7 from d-rec/developer_nm

Developer nm

NamrataSweya committed:
add enum

NamrataSweya committed:
add influxclient package

Ahmed-Suhail committed:
Merge pull request #8 from d-rec/developer_nm

Developer nm

NamrataSweya committed:
enumindex

Ahmed-Suhail committed:
Merge pull request #9 from d-rec/developer_nm

enumindex

NamrataSweya committed:
dremodule

Ahmed-Suhail committed:
Merge pull request #10 from d-rec/developer_nm

dremodule

Ahmed-Suhail committed:
suhail - updating device group schema and additional validation handling

Ahmed-Suhail committed:
suhail - updated lock file

NamrataSweya committed:
issue Certificate By frequency

change in function devicegroup  issue certificate of cron job

Ahmed-Suhail committed:
suhail - added log

Ahmed-Suhail committed:
suhail - updted role

Ahmed-Suhail committed:
suhail - updated influx to env variables

Ahmed-Suhail committed:
suhail - added ignore

Ahmed-Suhail committed:
suhail - added quotes

Ahmed-Suhail committed:
suhail - added quotes

Ahmed-Suhail committed:
suhail - added authorization access for meter read

Ahmed-Suhail committed:
suhail - updated

NamrataSweya committed:
add reject in aggregate

Ahmed-Suhail committed:
suhail - added authority to exceed and other required metadata for tracking certificate generation

Ahmed-Suhail committed:
Merge pull request #11 from d-rec/developer_nm

Developer nm

Ahmed-Suhail committed:
suhail - merged dev suhail and resolved conflicts

Ahmed-Suhail committed:
suhail - added authorityto exceed and other flags required in certitficate generation

NamrataSweya committed:
issuercertificate and buyerreservation

NamrataSweya committed:
cronjobissuercertificate

Ahmed-Suhail committed:
Merge pull request #12 from d-rec/developer_nm

Developer nm

Ahmed-Suhail committed:
suhail - adding conflict exception

Ahmed-Suhail committed:
suhail - updated device age

Ahmed-Suhail committed:
Merge branch 'dev_suhail' into developer_nm

NamrataSweya committed:
add list function of certificate group id

NamrataSweya committed:
change update next request certificate method

Ahmed-Suhail committed:
suhail - added left over reads by country code and remvoed if condition for checking same device

Ahmed-Suhail committed:
suhail - updated merge

Ahmed-Suhail committed:
suhail - added valdiation for history .. aggregate and delta

NamrataSweya committed:
meter read validation

NamrataSweya committed:
aggregate read validation

Ahmed-Suhail committed:
suhail - resolved issues in certificate generation

Ahmed-Suhail committed:
suhail - resolved merge conflict

NamrataSweya committed:
aggregatecommit

Ahmed-Suhail committed:
suhail - added if

Ahmed-Suhail committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

NamrataSweya committed:
remove reserved device  from reservation group

Ahmed-Suhail committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

Ahmed-Suhail committed:
suhail - updated end of reservation, authority to exceed options and fields renamed as Megawatthour

Ahmed-Suhail committed:
Merge pull request #13 from d-rec/developer_nm

Developer nm

NamrataSweya committed:
save date log of device and and and in cronjob

NamrataSweya committed:
add checklogdevice certificate

NamrataSweya committed:
migration device log

NamrataSweya committed:
device certificate get list

NamrataSweya committed:
update reservation api and add and remove

Ahmed-Suhail committed:
suhail - added nested validation for dto but not working need to add other package support

NamrataSweya committed:
check userligin on update

NamrataSweya committed:
add new log table for device group

Ahmed-Suhail committed:
suhail - added detailed error message when in conflict

Ahmed-Suhail committed:
suhail - added hourly in if

Ahmed-Suhail committed:
suhail - updated if for Delta reading

Ahmed-Suhail committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

NamrataSweya committed:
issuer service change

Ahmed-Suhail committed:
suhail - updated name

NamrataSweya committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

NamrataSweya committed:
country  code add in issuer

Ahmed-Suhail committed:
suhail - updated target related fields to double precision and made countryCode and commisionDate mandatory

Ahmed-Suhail committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

NamrataSweya committed:
migration update

NamrataSweya committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

NamrataSweya committed:
device group uuid

NamrataSweya committed:
add country list and add validation device registration

Ahmed-Suhail committed:
suhail - updated certificate structure and some validation updates in reservation and uid usage in reservation

Ahmed-Suhail committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

Ahmed-Suhail committed:
Merge branch 'dev_suhail' into developer_nm

Ahmed-Suhail committed:
Merge pull request #14 from d-rec/developer_nm

Developer nm

Ahmed-Suhail committed:
suhail - removed duplicate entry

Ahmed-Suhail committed:
removed 30 days static

Ahmed-Suhail committed:
suhail - date validation updated

Ahmed-Suhail committed:
suhail -  added brackets for query

Ahmed-Suhail committed:
suhail - aded staic expiration due to quotes issue

NamrataSweya committed:
change in ui and client api

NamrataSweya committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

NamrataSweya committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

Ahmed-Suhail committed:
Merge pull request #16 from d-rec/developer_nm

Developer nm

NamrataSweya committed:
add sample pages in ui

Ahmed-Suhail committed:
suhail - removed folder not used anymore

Ahmed-Suhail committed:
suhail - updated imports with dummy arrray

Ahmed-Suhail committed:
suhail - added api call

Ahmed-Suhail committed:
suhail - updated with notification

NamrataSweya committed:
device list add in ui

NamrataSweya committed:
new sample form

Ahmed-Suhail committed:
suhail - updated graph bar

Ahmed-Suhail committed:
suhail - added loader userEffect sample and type definition sample

Ahmed-Suhail committed:
suhail - added sample tables with api integration and sample popver

NamrataSweya committed:
add new column in check device log

Ahmed-Suhail committed:
suhail - removed deviceGroupDTO from certificate metadata

Ahmed-Suhail committed:
suhail - updated type as REC

Ahmed-Suhail committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

Ahmed-Suhail committed:
Merge pull request #17 from d-rec/developer_nm

Developer nm

Ahmed-Suhail committed:
suhail - merged dev_suhail

Ahmed-Suhail committed:
suhail - added version in certificate structure

Ahmed-Suhail committed:
suhail - changed cron to run at 5 minutes

Ahmed-Suhail committed:
suhail - made groupid even if sent null will be stored

Ahmed-Suhail committed:
suhail - updated cron to run eeveyr 10 minutes

NamrataSweya committed:
new history meater read table

NamrataSweya committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

Ahmed-Suhail committed:
suhail - updated date format of chains to be displayed

NamrataSweya committed:
new history log table

NamrataSweya committed:
new commit

NamrataSweya committed:
enumhistory

NamrataSweya committed:
add groupid

NamrataSweya committed:
new add history next log

NamrataSweya committed:
change in device group

Ahmed-Suhail committed:
Merge branch 'dev_suhail' into developer_nm

Ahmed-Suhail committed:
suhail - updated certitficate feilds

Ahmed-Suhail committed:
Merge pull request #18 from d-rec/developer_nm

Developer nm

Ahmed-Suhail committed:
suhail - removed extra index file

Ahmed-Suhail committed:
suhail - updated next issuance entry based on historical changes

NamrataSweya committed:
add new function history read

NamrataSweya committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

NamrataSweya committed:
add

Ahmed-Suhail committed:
suhail - added misssed sattement

Ahmed-Suhail committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

Ahmed-Suhail committed:
suhail - rechanged some lost changes

NamrataSweya committed:
add upddate function in issuer service ,read service

NamrataSweya committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

Ahmed-Suhail committed:
suhail - added next minimum

Ahmed-Suhail committed:
AhmeMerge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

Ahmed-Suhail committed:
suhail - updated next issuance with minimum

NamrataSweya committed:
ad caps device id

Ahmed-Suhail committed:
suhail - updated- query

Ahmed-Suhail committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

Ahmed-Suhail committed:
Merge pull request #19 from d-rec/developer_nm

Developer nm

Ahmed-Suhail committed:
suhail - changed default value from true to false

NamrataSweya committed:
add reads validation

Ahmed-Suhail committed:
Merge pull request #20 from d-rec/developer_nm

Developer nm

Ahmed-Suhail committed:
suhail - added validation

NamrataSweya committed:
update history device log and check history read service

NamrataSweya committed:
add enddate in issuer service

Ahmed-Suhail committed:
suhail - updated start date as previous minimum reading date, pending for aggreagate, and enddate as maximum current reading date ratehr than using currrent issuance cycle start and end time also added filtering of devices where there is no meter read

Ahmed-Suhail committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

Ahmed-Suhail committed:
suhail - added fitler for read type not found

Ahmed-Suhail committed:
suhail - added aggregate

NamrataSweya committed:
chnage innewissueCertificateForGroup function

NamrataSweya committed:
change in create function in groupservice

Ahmed-Suhail committed:
suhail - added ISO timestamp for certificate in UI

Ahmed-Suhail committed:
suhail - added missed line for minimumstart date

Ahmed-Suhail committed:
Merge branch 'developer_nm' of https://github.com/d-rec/drec-origin into developer_nm

Ahmed-Suhail committed:
suhail - fixed date issues when start time was not getting properly added to certificate and asynchronous push issue

Ahmed-Suhail committed:
Merge pull request #21 from d-rec/developer_nm

Developer nm

Ahmed-Suhail committed:
suhail - updated length check

Ahmed-Suhail committed:
suhail - fixed start dates logging in individual devices

NamrataSweya committed:
change datatype in DOUBLE PRECISION

NamrataSweya committed:
version change

Ahmed-Suhail committed:
Merge pull request #24 from d-rec/developer_nm

Developer nm

Ahmed-Suhail committed:
suhail - added emtory array for seed

Commit:
port changed for UI by Sandeep or Srinivas

Commit:
srinivas - added yaml files and buildscript

Commit:
Merge branch 'dev_suhail' of https://github.com/d-rec/drec-origin into dev_suhail

Ahmed-Suhail committed:
Merge pull request #25 from d-rec/developer_nm

suhail - added empty array for seed

NamrataSweya committed:
add new error handler for history validation

Ahmed-Suhail committed:
Merge pull request #27 from d-rec/developer_nm

add new error handler for history validation
